atom/common/platform_util_linux.cc

```diff
@@ -88,6 +89,14 @@ bool OpenExternal(const GURL& url, bool activate) {
     return XDGOpen(url.spec(), false);
 }
 
+bool OpenExternal(const GURL& url, bool activate,
+  const OpenExternalCallback& callback) {
```

Comment:
This line should either be aligned with the first function parameter above or indented by 4 spaces in total (whichever fits the ~80 column limit).